test/python/pulse/test_builder.py

```diff
@@ -367,6 +367,16 @@ def test_play_array_pulse(self):
 
         self.assertEqual(schedule, reference)
 
+    def test_play_name_argument(self):
```

Comment:
Would you mind adding a quick test of `name` setting to every `test_<inst>` test? I would really appreciate this.